admin.py

```diff
@@ -800,17 +800,41 @@ def BuildSensorList(self, software_version):
         recording |= 1
       if sensor.supports_min_max_recording:
         recording |= 2
+      try:
+          range_min=int(sensor.range_min)
+      except:
```

Comment:
These should be more specific so they only catch ValueError (and perhaps TypeError, which is thrown if you pass in None, i.e. the value isn't there to be queried). Otherwise you'll also catch and swallow more significant server errors like I'm a Teapot or I'm on Fire.

More generally, you raise an interesting question, as just above this we have the following code:
```
if personality.slot_count is not None:
  data['slot_count'] = int(personality.slot_count)
```

Which will fail in exactly the same way if someone submits a model collection where the slot_count field is foo.

So perhaps we need a little function that tries to run int() on a string, and either returns the number or None, which can also then wrap up checking if the string is None first.